### PR TITLE
feat: add healthcheck endpoint

### PR DESCRIPTION
**Pull request by pablocampogo**
Adds healthcheck endpoint and tests

### CODE REVIEW

File: api/server.go [hunk 1]
```diff
@@ -46,6 +46,14 @@ type ErrorResponse struct {
 	Message string
 }
 
+func healthCheck(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	_, err := w.Write([]byte("Pocket HTTP DB is up and running!"))
```
Comment (commoddity) [inline api/server.go]: should say Relay Meter

**Review by commoddity — CHANGES_REQUESTED** [review]
LGTM except the response string

File: api/server_test.go [hunk 2]
```diff
@@ -38,6 +38,12 @@ func TestGetHttpServer(t *testing.T) {
 		expectedStatusCode int
 		failAuth           bool
 	}{
+		{
+			name:               "Healthcheck",
+			url:                "http://relay-meter.pokt.network/",
```
Comment (rem1niscence) [inline api/server_test.go:43]: not too comfortable having this explicitly there, would rather have an env

**Review by rem1niscence — CHANGES_REQUESTED** [review]

File: api/server_test.go (hunk 2 above)
Comment (pablocampogo) [reply api/server_test.go:43]: This is already explicit on all test cases, we can add it as a betterment for later

**Review by rem1niscence — APPROVED** [review]

**Review by commoddity — APPROVED** [review]

**Pull request merged**